src/test/java/io/confluent/examples/streams/ApplicationResetIntegrationTest.java

```diff
@@ -18,13 +18,10 @@
 import io.confluent.examples.streams.kafka.EmbeddedSingleNodeKafkaCluster;
 import kafka.admin.AdminClient;
 import kafka.tools.StreamsResetter;
+import org.apache.kafka.clients.admin.AdminClientConfig;
 import org.apache.kafka.clients.consumer.ConsumerConfig;
 import org.apache.kafka.clients.producer.ProducerConfig;
-import org.apache.kafka.common.serialization.ByteArraySerializer;
-import org.apache.kafka.common.serialization.LongDeserializer;
-import org.apache.kafka.common.serialization.Serdes;
-import org.apache.kafka.common.serialization.StringDeserializer;
-import org.apache.kafka.common.serialization.StringSerializer;
+import org.apache.kafka.common.serialization.*;
```

Comment:
nit: avoid wildcard import (IntelliJ strikes again)

Reply:
Darn it. Thanks for the catch.